### ISSUE TITLE
Parse and transmit to the system more infos about satellites

### ISSUE THREAD

**Issue by kalon33** [freshollie/UsbGps4Droid#8]
Currently, when using an external device, we can see this kind of information in other GPS apps:
![screenshot_2017-08-27-18-55-34](https://user-images.githubusercontent.com/460731/29752193-eec77088-8b59-11e7-84fa-60f4f74758c5.png)
![screenshot_2017-08-27-18-55-31](https://user-images.githubusercontent.com/460731/29752194-eecd276c-8b59-11e7-9774-78e1db05803d.png)

This is what would be great to be displayed, as it is from the internal phone GPS device:
![screenshot_2017-08-27-18-56-14](https://user-images.githubusercontent.com/460731/29752208-15d8ba4c-8b5a-11e7-9c42-6bb0185bb9e5.png)
![screenshot_2017-08-27-18-56-10](https://user-images.githubusercontent.com/460731/29752209-15da546a-8b5a-11e7-9f76-eb484ded7f87.png)

Is this possible to have this kind of info? As previously, I'm ready to test and provide logs if needed :)

**Comment by freshollie** [comment]
No need for you to test. If I create this it will work with any NMEA data including my own GPS unit. Incidentally what is this app called? Is it open source because creating UI is the worst part, the data is easy to gather.

**Comment by kalon33** [comment]
https://github.com/barbeau/gpstest 

----- Original Message -----

**Comment by freshollie** [comment]
Wait, I just read the title and realised that I just need to send this info to the system. I will look into this.

**Comment by freshollie** [comment]
The only option would be for me to implement this into my app directly instead of an external app. I will be able to do this by potentially copying the code from this other app. I cannot send any of this data to android without root.

**Comment by freshollie** [comment]
I have started work on this

**Comment by sobrinho** [comment]
@freshollie have you ever finished this? :)

**Comment by freshollie** [comment]
@sobrinho Yeah it's mostly completed in its branch, however it has some problems which are mostly to do with the satellite position view. If you can build the APK it should run: https://github.com/freshollie/UsbGps4Droid/tree/satellite-analysis-development
